.licenserc.yaml

```diff
@@ -63,5 +63,6 @@ header:
     - '**/src/main/proto/protoc-gen-swagger/**'
     - '**/src/main/proto/validate/validate.proto'
     - '**/src/main/proto/opencensus/**'
+    - '**/*.textproto'
```

Comment:
This is incorrect move. 2 questions
1. Why `textproto` rather than proto? What format are we using?
2. Why don't add comments for ASF LICENSE header?

Reply:
The `textproto` is one of the `traceSeries` definitions in BanyanDB. This is used to determine the schema at runtime.

Reply:
What we did in BanyanDB, we doesn't care here. What is the schema, we doesn't care here. 

SkyWalking is a user for BanyanDB, we only care about the HTTP client APIs, and network protocol. That is all.
Keep this in mind, these are all we cared about. These are low level APIs.
https://github.com/apache/skywalking-banyandb/blob/b7845b0cd9fbebf71d3fc990ad6eae5f3ec16771/api/proto/banyandb/v1/rpc.proto#L29-L30